Send views data to influx (#185)
opendatateam/udata-piwik: CHANGELOG.md

## Current (in progress)

> - Changed metrics system. The are now wrote in influxDB [#185](https://github.com/opendatateam/udata-piwik/pull/185)

Author: ```suggestion
- Changed metrics system. They are now stored in influxDB [#185](https://github.com/opendatateam/udata-piwik/pull/185)
```

> ## 2.0.2 (2020-04-24)

Author: why is this in the diff? 🙈

## Current (in progress)

- Changed metrics system [#185](https://github.com/opendatateam/udata-piwik/pull/185):
  - Metrics are now stored into InfluxDB before being injected in udata's objects.
  - Udata piwik accesses influxDB throught [udata-metrics](https://github.com/opendatateam/udata-metrics)

>   - For the metrics injection, a periodic job needs to be scheduled, and a command was added to trigger it manually

Author: which job? which command?
